lib/transmogrify.js

```diff
@@ -25,15 +25,19 @@ module.exports = function (url, options, cb) {
     var deferred = q.defer();
 
     // transmogrify all the things
-    if (url.substr(0, 4) != 'http') {
-      // assume its a file if it starts with http
+    if (typeof url == 'string' && url.substr(0, 4) == 'http') {
```

Comment:
yarg takes care of reading the args where I'm actually using this. Unless you implicitly add whitespace by calling some app that uses transmogrify with `' http://...'` instead of `'http://...'` you won't get any whitespace.